npm/npm_import.bzl

```diff
@@ -675,6 +688,11 @@ def npm_import(
 
         **kwargs: Internal use only
     """
+
+    # TODO(2.0): move this to a new required rules_js_repositories() WORKSPACE function
```

Comment:
I dunno if we should ever bother, since we'll put effort into bzlmod migrations.

Reply:
it would be a small change one way or the other but yes I agree bzlmod is the future for all rulesets anyway